ducktape/utils/wait.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2015 Confluent Inc.
```

Comment:
My vote is keep this and ditch `wait_until`

As for time saved stuff... I guess it's useful if you're trying to convince people not to use wait.sleep, but ultimately do we really want to keep it around?